### DISCUSSION TITLE
What is this working group's purpose?

### DISCUSSION THREAD

**Discussion by kinlane** [OAI/sig-travel#2] [General]
Let's kick off the discussion board with a conversation about the purpose of this OAI travel group. Stu Waldron of OpenTravel has provided two documents to help jumpstart the discussion, and I'd like to see us move some of the conversations we've been having via email here to the discussion board to open up the discussion wider.

- [Open Travel & OAI Enabling Digital Travel Retail](https://docs.google.com/document/d/13aH7fLnOxjRPh1eDmVZHyKic-tmKakIU/edit)
- [OpenTravel OAI cooperation](https://docs.google.com/presentation/d/1bQ5TwJBmLi1Iw6qy77k-3zfGypdIidrN/edit#slide=id.p1)

Feel free to chime in here on this thread about the purpose, or make sure and join the Friday discussion via the weekly office hours.

**Comment by swaldron58** [comment]
Overall goal to lower the cost of publishing and consuming travel products and services via APIs. Digital retail in travel is held back by the chaos of bespoke APIs. There are hundreds of airlines to connect to, hundreds of rail, dozens of cruise lines, dozens of car rental, thousands of hotel companies (including home stays), and uncountable tour operators, restaurants, golf courses and others. Of all commercial activity considered related to travel (2019 in the US was $1.1T), only a tenth of the spend is accessible thru travel management apps or agents. it's too expensive to connect to the majority of travel due to API chaos. Lowering the costs will open up the market for all.

**Comment by swaldron58** [comment]
There are long existing travel standards and/or trade associations that are capable of representing their segment of travel (air, rail, hotel, car, tour operators ,,,,). The Open Travel Alliance has published since 2001 message standards for interoperability across the segments.  The proposal is to partner with OAI to move beyond message standards to API standards. A goal of this working group is to determine how that would work for the benefit of all.
